android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java

```diff
@@ -175,31 +334,83 @@ public void onResume() {
     }
 
     private void updatePreferences() {
+
+        for (int i = 0; i < NEW_PRIVACY_PREFERENCE_ORDER.length; i++) {
+            if (findPreference(NEW_PRIVACY_PREFERENCE_ORDER[i]) != null) {
+                findPreference(NEW_PRIVACY_PREFERENCE_ORDER[i]).setOrder(i);
+            }
+        }
         removePreferenceIfPresent(PREF_SYNC_AND_SERVICES_LINK);
+        String fingerprinting = BravePrefServiceBridge.getInstance().getFingerprintingControlType();
+        String cosmeticFiltering =
+                BravePrefServiceBridge.getInstance().getCosmeticFilteringControlType();
+        String ctlType = BravePrefServiceBridge.getInstance().getNoScriptControlType();
+        String cookiesBlockType = BravePrefServiceBridge.getInstance().getCookiesBlockType();
+        String getNoScriptControlType =
+                BravePrefServiceBridge.getInstance().getNoScriptControlType();
```

Comment:
not used var, and it's the same as `ctlType` which isn't used as well

Reply:
yes, i cleaned there, and i implemented the check for getNoScriptControlType(), which was missing...